source/testnet/net/installation/downloads.rst

```diff
@@ -110,14 +109,14 @@ Native Windows node
 
 To run a node on Windows, you need a Windows Installer package.
 
-- `Download the Windows Installer package<https://distribution.concordium.software/windows/Signed/Node-1.1.1.msi>`
+- `Download the Windows Installer package <https://distribution.concordium.software/windows/Signed/Node-1.1.1.msi>`_
 
 Native Mac node
 ----------------
 
 To run a node on macOS, you need a macOS installer package.
```

Comment:
```suggestion
To :ref:`run a node on macOS <run-node-macos>`, you need a macOS installer package.
```
And in the same way, a link to the macOS explanation can go here. For some reason it's not possible to comment on unchanged lines in PRs, but the same suggestion would go for the Linux nodes.